### PR TITLE
Add option to set ringtone for Groups

### PR DESCRIPTION
**Pull request by may-kumar**
Hi
I've tried adding an implementation of RingtonePicker to assign a custom ringtone across a Group.

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt [hunk 1]
```diff
@@ -136,4 +166,26 @@ class GroupContactsActivity : SimpleActivity(), RemoveFromGroupListener, Refresh
...
+            putExtra(RingtoneManager.EXTRA_RINGTONE_SHOW_SILENT, true)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_SHOW_DEFAULT, true)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_TYPE, RingtoneManager.TYPE_RINGTONE)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_DEFAULT_URI, defaultRingtoneUri)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_EXISTING_URI, defaultRingtoneUri)
+        }
+    }
+    private fun setRingtoneOnSelected(uri: Uri) {
+//        val contactRingtone = RingtoneManager.getRingtone(this, uri)
+//        group_ringtone.text = contactRingtone.getTitle(this)
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt:185]: whats with these 2 lines?

Comment (may-kumar) [reply app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt:185]: Sorry, forgot to remove those two commented lines, I was trying to make my own version, but then there was an implementation from Contacts which I reused.

File: app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt [hunk 2]
```diff
@@ -136,4 +166,25 @@ class GroupContactsActivity : SimpleActivity(), RemoveFromGroupListener, Refresh
...
+        return Intent(RingtoneManager.ACTION_RINGTONE_PICKER).apply {
+            putExtra(RingtoneManager.EXTRA_RINGTONE_SHOW_SILENT, true)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_SHOW_DEFAULT, true)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_TYPE, RingtoneManager.TYPE_RINGTONE)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_DEFAULT_URI, defaultRingtoneUri)
+            putExtra(RingtoneManager.EXTRA_RINGTONE_EXISTING_URI, defaultRingtoneUri)
+        }
+    }
+    private fun setRingtoneOnSelected(uri: Uri) {
+        //Assign ringtone to each contacts
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt:184]: remove this comment too please, it isnt necessary

Comment (may-kumar) [reply app/src/main/kotlin/com/simplemobiletools/contacts/pro/activities/GroupContactsActivity.kt]: Done!

**Comment by tibbi** [comment]
alright, thanks :)

**Pull request merged**